Packs/AnomaliThreatStreamFeed/Integrations/AnomaliThreatStreamFeed/AnomaliThreatStreamFeed.py

```diff
@@ -0,0 +1,544 @@
+import demistomock as demisto  # noqa: F401
+import urllib3
+from CommonServerPython import *  # noqa: F401
+from typing import Any
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+""" CONSTANTS """
+DEFAULT_MALICIOUS_THRESHOLD = 65
+DEFAULT_SUSPICIOUS_THRESHOLD = 25
+DEFAULT_BENIGN_THRESHOLD = 0
+THREAT_STREAM = "Anomali ThreatStream Feed"
+RETRY_COUNT = 2
+LIMIT = 1000
+STATUS = "active"
+
+indicatorType = {
+    "domain": FeedIndicatorType.Domain,
+    "ip": FeedIndicatorType.IP,
+    "md5": FeedIndicatorType.File,
+    "url": FeedIndicatorType.URL,
+    "email": FeedIndicatorType.Email,
+}
+
+RELATIONSHIPS_MAPPING = {
```

Comment:
If this map is a duplication of the map we have in ThreatStream V3, we need to consider taking it out somewhere to prevent code duplication. If that is the case, let's talk about it.